src/catalog/catalog_content.cpp

```diff
@@ -48,13 +73,11 @@ table_id_t CatalogContent::createNodeTable(const binder::BoundCreateTableInfo& i
     for (auto& propertyInfo : extraInfo->propertyInfos) {
         nodeTableEntry->addProperty(propertyInfo.name, propertyInfo.type.copy());
     }
-    tableNameToIDMap.emplace(nodeTableEntry->getName(), tableID);
-    tables->createEntry(std::move(nodeTableEntry));
-    return tableID;
+    return nodeTableEntry;
 }
 
-table_id_t CatalogContent::createRelTable(const binder::BoundCreateTableInfo& info) {
-    table_id_t tableID = assignNextTableID();
+std::unique_ptr<CatalogEntry> CatalogContent::createRelTableEntry(
+    common::table_id_t tableID, const binder::BoundCreateTableInfo& info) {
```

Comment:
using namespace kuzu::binder;
using namespace kuzu::common;
using namespace kuzu::storage;
I see these on the top of the file, so you can remove binder/common namespace there.

Reply:
good catch.